views/components/page_header.erb

```diff
@@ -2,7 +2,7 @@
 breadcrumbs ||= []
 back ||= breadcrumbs.dig(-2, 1) || "#" %>
 
-<div class="space-y-1">
+<div class="space-y-1 sticky -mx-10 mb-0.5 px-10 top-[2.9rem] bg-gray-50 z-30">
```

Comment:
My preference would be to not use sticky page headers, but I don't feel strongly about it.

Reply:
We can decide to remove the sticky page headers after we unify them. As it stands, some pages are sticky while others are not.